hathor/cli/run_node.py

```diff
@@ -49,6 +49,10 @@ def create_parser(self) -> ArgumentParser:
         parser.add_argument('--recursion-limit', type=int, help='Set python recursion limit')
         parser.add_argument('--allow-mining-without-peers', action='store_true', help='Allow mining without peers')
         parser.add_argument('--min-block-weight', type=int, help='Minimum weight for blocks')
+        parser.add_argument('--x-fast-init-beta', action='store_true',
+                            help='Execute a fast initialization, which skips some transaction verifications. '
+                            'This is still a beta feature because there might be some issues if there is a sudden'
+                            'crash on the full node.')
```

Comment:
This is still a beta feature as it may cause issues when restarting the full node after a crash .